common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -46,6 +46,12 @@ class InheritanceMixin(XBlockMixin):
         help=_("Enter the default date by which problems are due."),
         scope=Scope.settings,
     )
+    # This attribute is for custom pacing in self paced courses for Studio if CUSTOM_PLS flag is active
+    due_num_weeks = Integer(
+        display_name=_("Number of Weeks Due By"),
```

Comment:
Names are tough. Wish ya the best of luck here and feel free to crowdsource ideas.